packages/react-server-module-tagger/test.js

```diff
@@ -2,31 +2,31 @@ import test from 'ava';
 import loggerSpec from '.';
 
 test('creates a module tag', t => {
-  const expected = '{\"name\":\"foo.bar\",\"color\":{\"server\":73,\"client\":\"rgb(42,127,127)\"}}';
```

Comment:
Turns out just copy-pasting from the node terminal is _not_ the best way to get regression test values 😆